File: CC/java/micro07.java

public class micro07{

    public static void main(String[] args){
        int numero, programa;
        String opc;

        programa = 1;

        while(programa == 1){
            System.out.println("Digite um número: ");
            numero = Integer.parseInt(System.console().readLine());

            if(numero > 0 ) System.out.println("Positivo");
            else if (numero == 0) System.out.println("O número é igual a zero.");
            else System.out.println("Negativo");
        
            System.out.println("Deseja finalizar? (S/N)");
            opc = System.console().readLine();    

            if(opc.equals("S")) programa = 0;        
        }
    }
}

File: CC/trabalho8/final/final/exemplosLua/micro07.lua
-- Listagem 19: Decide se os números são positivos, zeros ou negativos

--[[ Função: Faça um algoritmo que receba N números e mostre positivo, negativo ou zero para cada número --]]

programa = 1
while (programa == 1)
do
	print("Digite um numero: ")
	numero = io.read()
	numero = tonumber(numero)

	if(numero > 0)
	then print("Positivo")
	elseif(numero == 0)
	then print("O número é igual a 0")
	elseif(numero < 0)
	then print("Negativo")
	end


	print("Deseja Finalizar? (S/N)")
	opc = io.read("*line")
	
	if(opc == "S")
	then programa = 0
	end
end